portfolio/src/main/java/com/google/sps/servlets/MyProjectsServlet.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.andrewrs.sps.FileReader;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/myProjects")
+public class MyProjectsServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    FileReader file = new FileReader("projects.html");
+    if(file.getData().length() > 0)
+    {
+        file = new FileReader("projects.html");
+    }
+    response.getWriter().println(file.getData());
+  }
+}
```

Comment:
It doesn't look like this servlet is used anywhere in your code. If it is, please show me (because I've been known to miss things like this in the past). If it isn't, please delete this whole file.

Reply:
it's not, but I was thinking I might use it later, but there is nothing interesting implemented there, I will remove it for my next pr

Reply:
pr7